.editorconfig

```diff
@@ -1,9 +1,13 @@
+# top-most EditorConfig file
+root = true
+
 # Unix-style newlines with a newline ending every file
 [*]
 end_of_line = lf
 insert_final_newline = true
 
-# 4 space indentation
-[*.py]
+# 4 space indentation and utf-8 charset
+[*.{rs, toml, yml}]
+charset = utf-8
```

Comment:
Let's keep yaml to 2 spaces, also move charset to the above global rule after `insert_final_newline`